src/genie/libs/parser/iosxe/show_key_chain.py

```diff
@@ -0,0 +1,123 @@
+'''
+show_key_chain.py
+Parser for following commands:
+    * show key chain
+'''
+
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Schema, Any, Or, Optional
+
+
+class ShowKeyChainSchema(MetaParser):
+    '''
+    Schema for
+        * show key chain
+    '''
+    schema = {
+        Any(): {
+            Any(): {
+                'key_string': str,
+                'accept_lifetime': {
+                    'start': str,
+                    'end': str,
+                    'is_valid': bool
+                },
+                'send_lifetime': {
+                    'start': str,
+                    'end': str,
+                    'is_valid': bool
+                },
+            },
+        },
+    }
```

Comment:
Since we have two `Any()` keys "together", please add in a legend to say what's what.
Ex
```
schema = {
    'key_chains': {
        Any(): {
            'keys': {
                Any(): {
                    'key_string': str,
                    'accept_lifetime': {
                        'start': str,
                        'end': str,
                        'is_valid': bool
                    },
                    'send_lifetime': {
                        'start': str,
                        'end': str,
                        'is_valid': bool
                    },
                },
            },
        },
    }
}
```